guide/src/migration.md

```diff
@@ -42,10 +42,11 @@ For more, see [the constructor section](https://pyo3.rs/master/class.html#constr
 PyO3 0.9 introduces [`PyCell`](https://pyo3.rs/master/doc/pyo3/pycell/struct.PyCell.html), which is
 a [`RefCell`](https://doc.rust-lang.org/std/cell/struct.RefCell.html) like object wrapper
 for dynamically ensuring
-[Rust's rule of Reference](https://doc.rust-lang.org/book/ch04-02-references-and-borrowing.html#the-rules-of-references).
+[Rust's rules of references](https://doc.rust-lang.org/book/ch04-02-references-and-borrowing.html#the-rules-of-references).
```

Comment:
I would be more explicit and say "rules regarding aliasing of references".

Reply:
Hmm I want to use the same expression as the rustbook here

Reply:
Sure, I just don't think "the rules of references" is a standing term in the community. I wouldn't immediately know what it refers to until clicking the link.

Reply:
Why not have both and write something like 

```
RefCell-like object wrapper for ensuring Rust's rules regarding aliasing of references are upheld.
For more detail, see the 
[Rust Book's section on Rust's rules of references](https://doc.rust-lang.org/book/ch04-02-references-and-borrowing.html#the-rules-of-references)
```